### PR TITLE
fix: several clarifications in the multiplatform testing tutorial to make the path smoother

### PR DESCRIPTION
**Pull request by zamulla**
The tutorial was reported as outdated, but nothing much has changed to prevent it from working. However, several clarity issues can make the impression of code that's not working.

### CODE REVIEW

File: topics/multiplatform-run-tests.md [hunk 1]
```diff
@@ -85,16 +83,18 @@ which has the [`kotlin.test`](https://kotlinlang.org/api/latest/kotlin.test/) AP
 3. The `commonTest` source set stores all common tests. Now you also need to create a corresponding folder in your project,
    which must have the same name.
 
-   Create a new directory in `shared/src`. Choose the `commonTest/kotlin` folder
-   from the list of standard options provided by the IDE:
+   Create a new directory in the `shared/src` folder.
```
Comment (koshachy) [inline topics/multiplatform-run-tests.md:86]: folder vs directory :)
If you refer to a project structure, use "directory".

File: topics/multiplatform-run-tests.md [hunk 2]
```diff
@@ -85,16 +83,18 @@ which has the [`kotlin.test`](https://kotlinlang.org/api/latest/kotlin.test/) AP
...
 
    ![Creating common test directory](create-common-test-dir.png){width=350}
 
-4. In the `kotlin` folder, create a new `common.example.search` directory.
+4. In the `commonTest/kotlin` folder, create a new `common.example.search` directory.
 5. In this directory, create the `Grep.kt` file and update it with the following unit test:
 
     ```kotlin
-    import kotlin.test.Test
+   package common.example.search
```
Comment (koshachy) [inline topics/multiplatform-run-tests.md:95]: It seems that something is wrong with the indentation of the code 🤔

File: topics/multiplatform-run-tests.md (hunk 1 above)
Comment (koshachy) [reply topics/multiplatform-run-tests.md:86]: If you refer to a regular OS file system, use "folder"

**Review by koshachy — CHANGES_REQUESTED** [review]
Some comments.

UPD: Sorry, haven't noticed that it is just a draft!

File: topics/multiplatform-run-tests.md (hunk 1 above)
Comment (zamulla) [reply topics/multiplatform-run-tests.md]: Done with folders, tell me if it looks consistent to you.

File: topics/multiplatform-run-tests.md [hunk 3]
```diff
@@ -191,8 +188,8 @@ version is optional. When the version is present, you only need the number at th
     }
     ```
 
-3. In the `commonTest/kotlin` folder, create a new `org.kmp.testing` directory.
-4. In this directory, create the `CurrentRuntimeTest.kt` file and update it with the following platform and framework-agnostic test:
+3. In the `commonTest/kotlin` folder, create a new `org.kmp.testing` package.
```
Comment (koshachy) [inline topics/multiplatform-run-tests.md:191]: In line 90, there is "directory"
In line 191, there is  "folder"
But they look pretty same 🤔

**Review by koshachy — APPROVED** [review]
One minor comment.

**Pull request merged**